packages/runc/runc.spec

```diff
@@ -14,6 +14,11 @@ Summary: CLI for running Open Containers
 License: Apache-2.0
 URL: https://%{goimport}
 Source0: https://%{goimport}/releases/download/v%{gover}/%{gorepo}.tar.xz#/%{gorepo}-v%{gover}.tar.xz
+Source1: https://%{goimport}/releases/download/v%{gover}/%{gorepo}.tar.xz.asc#/%{gorepo}-v%{gover}.tar.xz.asc
+# Aleksa Sarai
+Source2: gpgkey-B64E4955B29FA3D463F2A9062897FAD2B7E9446F.asc
+# Kir Kolyshkin
+Source3: gpgkey-C2428CD75720FACDCF76B6EA17DE5ECB75A1100E.asc
```

Comment:
I can remove the unused key, but my reasoning for keeping both is that if each release is signed by one of these two, there will be less churn overall to list both and just switch which one is used for verification.

Reply:
Sounds fair - we can always change our mind on that as we see how it evolves!